RadixWallet/Clients/NPSSurveyClient/NPSSurveyClient+Live.swift

```diff
@@ -0,0 +1,151 @@
+extension DependencyValues {
+	public var npsSurveyClient: NPSSurveyClient {
+		get { self[NPSSurveyClient.self] }
+		set { self[NPSSurveyClient.self] = newValue }
+	}
+}
+
+// MARK: - NPSSurveyClient + DependencyKey
+extension NPSSurveyClient: DependencyKey {
+	public static var liveValue: NPSSurveyClient {
+		@Dependency(\.userDefaults) var userDefaults
+
+		let shouldAskForFeedbackSubject: AsyncCurrentValueSubject<Bool> = .init(false)
+
+		return .init(
+			uploadUserFeedback: { feedback in
+				await uploadUserFeedback(feedback)
+				shouldAskForFeedbackSubject.send(false)
+			},
+			incrementTransactionCompleteCounter: {
+				let currentCounter = userDefaults.getTransactionsCompletedCounter() ?? 0
+				let updatedCounter = currentCounter + 1
+				userDefaults.setTransactionsCompletedCounter(updatedCounter)
+				if Self.shouldAskUserForFeedback(updatedCounter) {
+					shouldAskForFeedbackSubject.send(true)
+				}
+			},
+			shouldAskForUserFeedback: {
+				shouldAskForFeedbackSubject.eraseToAnyAsyncSequence()
+			}
+		)
+	}
+}
+
+extension NPSSurveyClient {
+	private static let feedbackTransactionCounterThreshold = 10
+	private static let nextFeedbackIntervalThresholdInMonths = 3
+
+	@Sendable
+	static func shouldAskUserForFeedback(_ transactionCounter: Int) -> Bool {
+		@Dependency(\.userDefaults) var userDefaults
+		@Dependency(\.date) var date
+
+		if transactionCounter > Self.feedbackTransactionCounterThreshold {
```

Comment:
this is wrong right?

you have the same condition in both if and else if being `transactionCounter > Self.feedbackTransactionCounterThreshold`.

Reply:
should be `==`, forgot to remove `>` as it was used for quick testing... good catch